packages/altfire_configurator/lib/src/config.dart

```diff
@@ -5,19 +5,19 @@ import 'dart:async';
 class Config<T> {
   Config({
     required T value,
-    required StreamSubscription<void> subscription,
+    required StreamSubscription<void>? subscription,
```

Comment:
I thought `this.subscription` would be fine, but I'm wondering if you're purposely assigning it to `_subscription` to keep it from being accessed externally.

```suggestion
    this.subscription,
```

Reply:
I was probably intending that!